code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -473,7 +480,7 @@
 		return FALSE
 	limb = newBodyPart(limb_zone, 0, 0)
 	if(limb)
-		if(!limb.attach_limb(src, 1))
+		if(!limb.try_attach_limb(src, 1))
```

Comment:
True instead of 1?

Reply:
the var on whether I did your mom or not is set to 1

Reply:
GOTTEM